lib/web_ui/lib/src/engine/window.dart

```diff
@@ -36,12 +28,10 @@ class EngineWindow extends ui.Window {
 
   /// Overrides the default device pixel ratio.
   ///
-  /// This is useful in tests to emulate screens of different dimensions.
-  void debugOverrideDevicePixelRatio(double value) {
-    assert(() {
-      _debugDevicePixelRatio = value;
-      return true;
-    }());
+  /// This is useful in tests to emulate screens of different dimensions
+  /// including integration tests.
+  void overrideDevicePixelRatio(double value) {
+    _debugDevicePixelRatio = value;
```

Comment:
Why this change? It's still used only in tests, so the "debug" prefix and `assert` seem appropriate.

Reply:
It is now also used in integration test (the one that loads an image at framework level).

Reply:
Right, but that's still a test :)

Reply:
Do integration tests strip out asserts?

Reply:
If they don't, we should add the option to enable them. `dart2js` supports asserts (we use `dart2js` for engine unit-test with assertions enabled).

/cc @nturgut

Reply:
My comment is non-blocking though. I don't think there's major risk. It just doesn't feel right for a method that's not annotated as "debug" to manipulate a "debug" field without an assertion.